src/arutils.py

```diff
@@ -437,6 +456,8 @@ def polyval2d(x, y, m):
         z += a * x**i * y**j
     return z
 
+def gauss_3deg(x,ampl,cent,sigm):
+    return ampl*np.exp(-1*(cent-x)**2/2/sigm**2)
```

Comment:
Given that it's a straightforward code replacement (I think), perhaps we should explicitly test this with the object spectrum you sent me the other day. It would be interesting to see the difference between the two object profiles, i.e. a plot of `(gauss_3deg - gfunc)/gauss_3deg`, and the difference between the two "optimal" spectra.

I'm more tempted to use the integrated Gaussian (i.e. of the form in `gfunc`), since the detector pixels are technically integrating over the object profile.

Suppose someone resolves the object profile with 2 pixels FWHM, then `gauss_3deg` effectively represents the curvature of a Gaussian near the peak (where most of the flux is), with two straight lines.

What I'm not sure about, is how this changes the object profile. In the case of low S/N data, I would think this might be more of a problem. But I agree that this might be a non-issue. Is there a benefit of using something like `gauss_3deg` over something like `gfunc()`? e.g. a significant speedup?